src/interfaces/chain.h

```diff
@@ -135,6 +132,9 @@ class Chain
     //! included in the current chain.
     virtual Optional<int> getBlockHeight(const uint256& hash) = 0;
 
+    //! Get block hash. Height must be valid or this function will abort.
+    virtual uint256 getBlockHash(int height) = 0;
```

Comment:
In commit "[wallet] Move getBlockHash from Chain::Lock interface to simple Chain" (c6fa030ca8a282a7906260186e93d8c8902aa5a4)

Note: After this commit, the only remaining Chain::getBlockHash() calls are in CreateWalletFromFile and are removed in #15719. I expect this PR will get merged before #15719, so just making a note about this.

Reply:
Yes I have also #17484, #17443 which remove more interface methods to skim in after.